crates/holochain/src/conductor/conductor.rs

```diff
@@ -2601,17 +2601,33 @@ mod service_impls {
             let derivation_path = [0].into();
             let dst_tag = format!("{device_seed_lair_tag}.0");
 
-            let seed_info = self
-                .keystore()
-                .lair_client()
-                .derive_seed(
-                    device_seed_lair_tag.clone().into(),
-                    None,
-                    dst_tag.into(),
-                    None,
-                    derivation_path,
-                )
-                .await?;
+            let entry_info = self.keystore().lair_client().get_entry(dst_tag.clone().into()).await;
```

Comment:
Shouldn't this be, ?

```suggestion
            let entry_info = self.keystore().lair_client().get_entry(device_seed_lair_tag.clone().into()).await;
```